Instagram/Instagram/NotificationPage/IsOnInstViewCell.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  IsOnInstViewCell.swift
+//  Instagram
+//
+//  Created by Владислав Лымарь on 19.10.2022.
+//
+
+import UIKit
+
+/// Настройка ячейки "Есть в инстаграм"
+final class IsOnInstViewCell: UITableViewCell {
```

Comment:
```suggestion
final class OnInstViewCell: UITableViewCell {
```
Is такая приставка не подходит